pennylane/transforms/unbroadcast_expand.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains the tape expansion function for unbroadcasting a
+broadcasted tape into multiple tapes."""
+import pennylane as qml
+from pennylane.transforms.batch_transform import batch_transform
+
+
+@batch_transform
+def unbroadcast_expand(tape):
+    r"""Expand a broadcasted tape into multiple unbroadcasted tapes
+    and a function that stacks and squeezes the results.
+
+    Args:
+        tape (.QuantumTape): Broadcasted tape to be expanded
+    Returns:
+        list[.QuantumTape]: Unbroadcasted tapes that produce one of the
+        results of the broadcasted tape each
+        callable: Function that stacks the results
```

Comment:
Might need to double check this renders correctly once this shows up in the docs!

Reply:
It didn't. Changed the formatting to a tuple.